docs/src/pages/guides/aliases.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Aliases
+---
+
+An alias is a handy shortcut to your paths. Projects can get quite messy when it comes to file structure. Astro gives you the ability to define _keywords_ which you can use no matter how deep your file is located. 
+
+So instead of `../../components/SomeComponent/` you can use `components/SomeComponent`. 
```

Comment:
nit: don't start a sentence with So if you can help it!

```suggestion
With aliases, you can import from `"components/SomeComponent.astro"` instead of `"../../../../../components/SomeComponent.astro"`.
```